File-Formats/DocIO/Convert-Word-Document-to-PDF-in-ASP-NET-MVC.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Convert Word document to PDF in ASP.NET MVC | Syncfusion
+description: Convert Word document to PDF without Microsoft Word or interop dependencies in ASP.NET MVC application using .NET Word (DocIO) library.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word document to PDF in ASP.NET MVC
+
+Syncfusion Essential DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **convert a Word document to PDF in ASP.NET MVC**.
+
+## Steps to convert word document to PDF in C#:
```

Comment:
word document should always be "Word document"

Reply:
Updated sir.